### DISCUSSION TITLE
[Help] Why start Worldserver.exe there is lot of errors please?

### DISCUSSION THREAD

**Discussion by DevGamX** [TrinityCore/TrinityCore#28618] [Q&A] [answered]
Hello, 

I run the server on my computer with Windows 11, Intel 7 and 32 GB of memory

I compiled with VS 2022 and VS 2019 and got no errors.

I installed MySQL 8.0.30 and installed TDB 1002.22121 (master = 10.0.2.47213) with no issues with updates. 

Then I made extractor dbc, camera, gt, map, mmap, 
vmap from client Battle.net (master = 10.0.2.47213) and put in ClientData and gave DataDir path in worldserver.conf (DataDir = ".\ClientData")

But when I run worldserver.exe I only get errors? 

Frankly I'm at my wit's end and I really don't understand why all these errors.

Can you clarify the problem for me please? 

(I have the same problem with 9.2.7 45745)

Thank you so much!
[DBErrors.zip](https://github.com/TrinityCore/TrinityCore/files/10285993/DBErrors.zip)

![image](https://user-images.githubusercontent.com/118358135/209116964-5060a3e1-cb56-41e1-901f-6c4fa76d9d7c.png)

**Comment by Jinnaix** [comment, marked as answer]
Thats to be expected and not something you did wrong on your end.
If you want something in a playable state you go for 335 branch.
If you want something to work on, you can see master needs some love.

**Reply by DevGamX** [reply]
Ok, thank you Jinnaix, I'll take note of it, but I would have liked to open a Shadowlands server locally.
But yet for him (youtube video) it works fine: https://www.youtube.com/watch?v=SmEREpcO2rU

**Reply by Jinnaix** [reply]
Feel free to ignore that.
That video is 
- edited (look how worldserver is up and running on 0:29 but it states it took 1 minute and 25 seconds to boot)
- based on a repack that is or was closed source and users had to pay for it (if i am not mistaken)
- probably not showing/logging DBErrors (you could do that too to make it look fine, wont change the outcome that stuff is still missing)

Some people rather do work on their own thing to generate money with it instead of doing it for open source projects.

**Reply by DevGamX** [reply]
Ah, I understand better, thank you very much Jinnaix for your very clear explanations. Indeed if we can't launch and work properly on Trinicore Shadowlands, I'm going to get closer to 3.3 5. In his video we actually see 1 minute 25 in my opinion the errors are hidden as you say and yes it's a shame that he charges for his repacks.

**Comment by DevGamX** [comment]
I configured the parameters below and indeed I no longer have any errors like in the video (lol). So the problem comes from version 9.2.7 or 10 which is buggy or not complete.

So I have my answer and thank you to a moderator for kindly closing my topic.

Thank you again for your advice Jinnaix which enlightened me.

----------------------------------------------------
Logger.root=0,Console Server
Logger.server=3,Console Server
Logger.commands.gm=0,Console GM
Logger.scripts.hotswap=0,Console Server
Logger.sql.sql=0,Console DBErrors
Logger.sql.updates=0,Console Server
Logger.mmaps=0,Server
----------------------------------------------------

**Comment by Wraithxtc** [comment]
I'd like to add something since I went through this myself back in April of 2022.  Always go straight to the source when you want to learn everything about it. Let's just say, there are a lot of skiddies and scammers making guides these days.
